ads.properties.sample

```diff
@@ -0,0 +1,49 @@
+# OAuth refresh token
```

Comment:
Add an intro comment that explains what this file is and points to the [Getting started guide for Java](https://developers.google.com/google-ads/api/docs/client-libs/java/getting-started).

Also, friendly reminder to update that guide to point to this sample file if you think that'd be useful.

Reply:
Added to the intro and mailed cl/334568432